Models/Vehicle.cs

```diff
@@ -10,5 +10,8 @@ public class Vehicle
         public float batteryLife { get; set; }
         public double lastUpdated { get; set; }
         public Coordinate currentPosition { get; set; }
+        public bool fireFound { get; set; }
```

Comment:
fireFound and fireCoordinate should be optional properties, not required properties